python/lsst/meas/deblender/deblend.py

```diff
@@ -869,26 +867,24 @@ def _addChild(self, parentId, peak, sources, heavy):
         return src
 
     @pipeBase.timeMethod
-    def deblend(self, exposures, sources, psfs, bands=None):
+    def deblend(self, mExposure, sources, psfs):
         """Deblend a data cube of multiband images
 
         Parameters
         ----------
-        exposures: dict
-            Keys of the dict are the names of the filters and values are
-            `lsst.afw.image.exposure.exposure.ExposureF`'s.
+        mExposure: `MultibandExposure`
             The exposures should be co-added images of the same
             shape and region of the sky.
         sources: dict
-            Keys are the names of the filters and the values are
-            `lsst.afw.table.source.source.SourceCatalog`'s, which
-            should be a merged catalog of the sources in each band ('deepCoadd_mergeDet').
+            Keys are the names of the filters
+            (should be the same as `mExposure.filters`) and the values are
+            `lsst.afw.table.source.source.SourceCatalog`'s,which
```

Comment:
Space after comma? Is there a check that they are correct names?

Reply:
There is now...